packages/compiler-core/src/transforms/hoistStatic.ts

```diff
@@ -242,10 +251,16 @@ function hasDynamicKeyOrRef(node: ElementNode): boolean {
   return !!(findProp(node, 'key', true) || findProp(node, 'ref', true))
 }
 
-function hasCachedProps(node: PlainElementNode): boolean {
+function hasCachedProps(
+  node: PlainElementNode,
+  context: TransformContext
```

Comment:
since the other functions are putting `context` first, wouldn't it make more sense to also put the `context` argument first here?

Reply:
I think it is doesn't matter :). The code base has other usage with same...

Reply:
It does make the code more consistent and more intuitive to work with if functions consuming similar objects use the same order of arguments.

It's worth refactoring for a more consistent internal usage but I agree other changes unrelated to what you are fixing do not belong to this PR